formats/proxmox.nix

```diff
@@ -1,7 +1,16 @@
-{modulesPath, ...}: {
+{
+  modulesPath,
+  specialArgs,
+  ...
+}: {
   imports = [
     "${toString modulesPath}/virtualisation/proxmox-image.nix"
   ];
+
+  proxmox.qemuConf.diskSize =
+    if builtins.hasAttr "diskSize" specialArgs then specialArgs.diskSize
+    else "auto";
```

Comment:
proxmox.qemuConf.diskSize = specialArgs.diskSize or "auto"

Reply:
That is much nicer, thanks for the pointer.